dev-tools/packaging/templates/ironbank/elastic-agent/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Extract Elastic Agent and make various file manipulations.
 ################################################################################
 ARG BASE_REGISTRY=registry1.dsop.io
-ARG BASE_IMAGE=ironbank/redhat/ubi/ubi8
+ARG BASE_IMAGE=redhat/ubi/ubi9
```

Comment:
This change in addition to the `dev-tools/packaging/templates/ironbank/elastic-agent/hardening_manifest.yaml` are new since 7.17 does use elastic-agent